### ISSUE TITLE
`module not found in search path` error running the latest focal image

### ISSUE THREAD

**Issue by vkarak** [flux-framework/flux-sched#1068]
Our [CI](https://github.com/reframe-hpc/reframe/actions/runs/6139539304/job/16657544628?pr=2992#step:6:10) failed recently with the latest `focal` image `fluxrm/flux-sched:focal`. I could reproduce the error outside the CI as follows:

```bash
docker pull fluxrm/flux-sched:focal
docker run -it fluxrm/flux-sched:focal
```

```console
Sep 11 20:21:39.799984 broker.err[0]: rc1.0: flux-module: load sched-fluxion-resource: module not found in search path
Sep 11 20:21:39.802094 broker.err[0]: rc1.0: /etc/flux/rc1 Exited (rc=1) 0.2s
```

The previous version `fluxrm/flux-sched:focal-v0.28.0` works fine. Am I doing something wrong or is there indeed a problem with the latest published image?

**Comment by grondo** [comment]
I think this was an issue with the published docker images, fixed by #1100

**Comment by vkarak** [comment]
Thanks, we will try a later one.
